db/version_set.h

```diff
@@ -135,6 +135,14 @@ class VersionStorageInfo {
   // ComputeCompactionScore()
   void ComputeFilesMarkedForCompaction();
 
+  // This computes bottommost_files_marked_for_compaction_ and is called by
+  // ComputeCompactionScore() or UpdateOldestSnapshot().
+  // Among bottommost files (assumes they've already been computed), marks the
+  // ones that have keys that would be eliminated if recompacted, according to
+  // the seqnum of the oldest existing snapshot.
+  // REQUIRES: DB mutex held
```

Comment:
Nit: I would add a comment saying that since its behavior depends on oldest_sequence_number_ it has to be called every time oldest_sequence_number_ changes.

Reply:
Sure.